src/css/_largeCSS.scss

```diff
@@ -0,0 +1,34 @@
+// Media Query for screen size over 1500px;
+@media (min-width: 1500px) {
```

Comment:
These look great, I haven't had a chance to test it out on an actual big screen so I'm looking forward to that.